db/schema.rb

```diff
@@ -509,6 +509,7 @@
   add_index "listing_units", ["listing_shape_id"], name: "index_listing_units_on_listing_shape_id", using: :btree
 
   create_table "listings", force: :cascade do |t|
+    t.binary   "uuid",                            limit: 16
```

Comment:
This will most likely be a problem. Using the `:binary, limit: 16` type in the migration generated a column of type `VARBINARY`, which is not what we want. However, when setting up a new DB from this schema, I expect Rails to create a `VARBINARY` column again, assuming that the `:binary` -> `VARBINARY` mapping always holds.